src/canvas-extractor/edfi_canvas_extractor/extract_graphql.py

```diff
@@ -75,10 +109,13 @@ def run(arguments: MainArguments) -> None:
 
     for account in accounts:
         _id = getattr(account, "id")
```

Comment:
Let's add a logging statement to help us know which accounts are available:

```suggestion
        _id = getattr(account, "id")
        logger.info(f"Getting data for account ID {_id}")
```